angular-ngrx-scss/src/app/state/user/user.state.ts

```diff
@@ -10,6 +12,8 @@ export interface UserState {
   name: string;
   twitter_username: string;
   username: string;
+  repos?: UserReposState[];
```

Comment:
I think this needs to be renamed to topRepos. It's possible the full list of repos will also be saved here, and that will help us know which is which and not accidentally overwrite them.

- [ ] Rename to topRepos